BossMod/AI/AIController.cs

```diff
@@ -20,7 +20,7 @@ sealed class AIController(ActionManagerEx amex, MovementOverride movement)
     private DateTime _nextInteract;
     private DateTime _nextJump;
 
-    public bool InCutscene => Service.Condition[ConditionFlag.OccupiedInCutSceneEvent] || Service.Condition[ConditionFlag.WatchingCutscene78] || Service.Condition[ConditionFlag.Occupied33] || Service.Condition[ConditionFlag.BetweenAreas] || Service.Condition[ConditionFlag.OccupiedInQuestEvent];
+    public static bool InCutscene => Service.Condition[ConditionFlag.OccupiedInCutSceneEvent] || Service.Condition[ConditionFlag.WatchingCutscene78] || Service.Condition[ConditionFlag.Occupied33] || Service.Condition[ConditionFlag.BetweenAreas] || Service.Condition[ConditionFlag.OccupiedInQuestEvent];
```

Comment:
What is it used by now? Can't see any new references... Maybe it needs to be moved somewhere else?